docs/samples/contact-center/app.js

```
@@ -276,7 +276,13 @@ function register() {
       taskEvents.detail.task = task;
 
       incomingCallListener.dispatchEvent(taskEvents);
-    })
+    });
+
+    webex.cc.on('AgentStateChange', (data) => {
+      if(data.type = 'AgentStateChangeSuccess') {
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)